packaging_apps_git_fr.md

```diff
@@ -1,72 +1,72 @@
 # Comment utiliser Git pour packager les applications
 
-Git... Notre cher Git bien-aimé, que l'on peut aussi décrire comme "Goddamn Idiotic Truckload of sh*t", selon Linus.  
+Git... Notre cher Git bien-aimé, que l'on peut aussi décrire comme "Goddamn Idiotic Truckload of sh*t"  ( P *t**n de camion de m*rde), selon Linus.
 Si vous ne connaissez pas encore Git, soyez sûr que vous serez bientôt d'accord avec cette description.
 
-YunoHost et toutes nos applications sont sur la forge Git GitHub. Ce qui veut dire que si vous voulez travailler sur une application, tôt ou tard vous allez devoir faire face à Git.  
```

Comment:
I was wondering why you did have to add back the double space at the end of the lines.
But actually, that's the opposite !

Those double space are here in purpose, so it adds a new line.

Reply:
I didn't add double lines, I was wondering why they were there, but I didn't touch them. For information I use Atom for writing, maybe this software change this ?

Reply:
Probably the common auto correction of removing trailing spaces at the end of lines.
Better to keep them.

Reply:
I trie to mchange parameters